src/borg/archiver.py

```diff
@@ -1180,7 +1182,13 @@ def _delete_archives(self, args, repository):
                 else:
                     deleted = True
                     msg = 'Would delete: {} ({}/{})' if dry_run else 'Deleted archive: {} ({}/{})'
-                    logger.info(msg.format(format_archive(current_archive), i, len(archive_names)))
+                    if output_list:
+                        logger_list.info(msg.format(format_archive(current_archive),
+                                                    i, len(archive_names)))
+                    else:
+                        logger.info(msg.format(format_archive(current_archive),
+                                               i, len(archive_names)))
+
```

Comment:
this is still outputting the (same) list even when not giving `--list`.

and while you're at it: if `msg` is only used in the if-block, move it there.

Reply:
also: is it `self.output_list` ?

Reply:
yup, the tests also found that ^^^.